main.js

```diff
@@ -32,9 +32,19 @@ var startNewActivityForm = document.querySelector('.start-new-activity')
 var createNewActivityButton = document.querySelector('.create-new-activity-button')
 var userInputs = document.querySelectorAll('input')
 
+
+window.addEventListener('load', function() {
+  if (localStorage.length === 0) {
+    return
+  } else {
+    windowLoad()
+    addClass(defaultMessage)
+  }
+})
+
```

Comment:
Nice! I noticed that when you add enough activities we get white space at the bottom of the page

Reply:
Great thinking on testing that design component!